src/devices/node/node.cpp

```diff
@@ -14,26 +16,86 @@
 
 const int kLedPin = 0;
 
+constexpr DeviceMode kDeviceMode = DeviceMode::READ_FROM_FLASH;
+
 // Note: `RadioHeadRadio` needs to be a pointer - if it's an object, the node
 // crashes upon receiving a packet.
 RadioHeadRadio *radio = new RadioHeadRadio();
 NetworkManager nm(radio);
 RadioStateMachine state_machine(&nm);
-FastLedManager led_manager(Devices::current, &state_machine);
+FastLedManager *led_manager;
+
+constexpr uint32_t kEepromStart = 0x00020000 - 0x2000;
+FlashClass device_storage(/*flash_addr=*/(void *)kEepromStart,
+                          /*size=*/DeviceDescription::kMaxSize);
 
 void FeedWatchdog() { WDT->CLEAR.reg = WDT_CLEAR_CLEAR_KEY; }
 
+FastLedManager *ReadDeviceFromFlash() {
```

Comment:
This appears to be repeated, can we DRY the function up?

Reply:
The APIs for flash storage between the two platforms are just different enough that I think it makes sense to have this repeated.